docs/linux/setup.md

```diff
@@ -27,18 +27,14 @@ If you encounter any troubles, check the [troubleshooting](/docs/troubleshooting
 
 ### Go and syzkaller
 
-`syzkaller` is written in [Go](https://golang.org), and `Go 1.13+`
-toolchain is required for build. *Note:* `Go 1.14` is required for contributors,
-as `Go 1.13` may change `go.mod` file. The toolchain can be installed with:
+`syzkaller` is written in [Go](https://golang.org), and `Go 1.16+` toolchain is required for build.
+Generally we aim at supporting 2 latest releases of Go.
+The toolchain can be installed with:
 
 ```
-wget https://dl.google.com/go/go1.14.2.linux-amd64.tar.gz
-tar -xf go1.14.2.linux-amd64.tar.gz
-mv go goroot
-mkdir gopath
-export GOPATH=`pwd`/gopath
-export GOROOT=`pwd`/goroot
-export PATH=$GOPATH/bin:$PATH
+wget https://dl.google.com/go/go1.17.6.linux-amd64.tar.gz
+tar -xf go1.17.6.linux-amd64.tar.gz
+export GOROOT=`pwd`/go
```

Comment:
Just curious - we're targeting the engineers w/o the Go experience. Considering that point - is it better to specify GOPATH, or omit it?..

Reply:
GOPATH is a rudiment in the modules world and more steps to do. So I think for users we should describe normal 'git clone' + modules:
https://groups.google.com/g/syzkaller/c/T9iGEH8WiAo/m/3CdQgJlQBgAJ